actioncable/lib/action_cable/engine.rb

```diff
@@ -2,11 +2,14 @@
 require "action_cable"
 require "action_cable/helpers/action_cable_helper"
 require "active_support/core_ext/hash/indifferent_access"
+require "action_dispatch/routing/inspector"
+require "action_cable/route_wrapper"
 
 module ActionCable
   class Railtie < Rails::Engine # :nodoc:
     config.action_cable = ActiveSupport::OrderedOptions.new
     config.action_cable.url = '/cable'
+    config.action_cable.mounted = true
```

Comment:
Should we mention `/cable`? While it is the default, it's also configurable via `url`.